index.html

```diff
@@ -353,12 +353,15 @@ <h3>
                 <a>default permission query algorithm</a>.
               </p>
               <p>
-                The <dfn>default permission query algorithm</dfn>, given a {{PermissionDescriptor}}
-                <var>permissionDesc</var> and a {{PermissionStatus}} |status|, runs the following
+                The <dfn>default permission query algorithm</dfn>, given a to |permissionDesc|'s
+                |permissionDesc:PermissionDescriptor| and a {{PermissionStatus}} |status:PermissionStatus|, runs the following
```

Comment:
Something's wrong with the grammar here. Maybe

```suggestion
                The <dfn>default permission query algorithm</dfn>, given a {{PermissionDescriptor}}
                |permissionDesc:PermissionDescriptor| and a {{PermissionStatus}} |status:PermissionStatus|, runs the following
```

?

Reply:
yeah, that's better.